### ISSUE TITLE
irsync fails if file uid / gid not mappable to username / groupname

### ISSUE THREAD

**Issue by burnce** [irods/irods#2513]
`irsync` fails if the uid of a file cannot be translated to a username:

```
bash-4.1$ irsync -r . i:/home/user/directory
ERROR: getUnixUsername: no user with uid 15046
ERROR: copyFileMetaFromStat: could not retrieve username for uid 15046
ERROR: rsyncUtil: resolveRodsTarget status = -122000 SYS_USER_RETRIEVE_ERR

bash-4.1$ irsync -r . i:/home/user/directory
ERROR: getUnixGroupname: error calling getgrgid for gid 505. errno = 2
ERROR: copyFileMetaFromStat: could not retrieve groupname for gid 505
ERROR: rsyncUtil: resolveRodsTarget status = -131002 SYS_GROUP_RETRIEVE_ERR, No such file or directory
```

Why is the ownership needed? It would be logical that with no added flags, the user logged in to iRODS would own the synced files. Flags to explicitly set ownership mapping semantics would be helpful.

**Comment by tempoz** [comment]
These commits should fix this issue in 4.1

**Comment by kellerb** [comment]
This issue includes the removal of filesystem_meta.
